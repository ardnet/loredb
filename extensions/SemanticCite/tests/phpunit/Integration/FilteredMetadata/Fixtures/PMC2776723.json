{
    "header": {
        "type": "esummary",
        "version": "0.3"
    },
    "result": {
        "uids": [
            "2776723"
        ],
        "2776723": {
            "uid": "2776723",
            "pubdate": "2008/05/16 00:00",
            "epubdate": "",
            "printpubdate": "2008/05/16 00:00",
            "epubdate": "1/01/01 00:00",
            "source": "Science",
            "authors": [
                {
                    "name": "Bromberg KD",
                    "authtype": "Author"
                },
                {
                    "name": "Ma'ayan A",
                    "authtype": "Author"
                },
                {
                    "name": "Neves SR",
                    "authtype": "Author"
                },
                {
                    "name": "Iyengar R",
                    "authtype": "Author"
                }
            ],
            "title": "Design Logic of a Cannabinoid Receptor Signaling Network That Triggers Neurite Outgrowth",
            "volume": "320",
            "issue": "5878",
            "pages": "903-909",
            "articleids": [
                {
                    "idtype": "pmid",
                    "value": "18487186"
                },
                {
                    "idtype": "doi",
                    "value": "10.1126/science.1152662"
                },
                {
                    "idtype": "pmcid",
                    "value": "PMC2776723"
                },
                {
                    "idtype": "MID",
                    "value": "NIHMS142954"
                }
            ],
            "fulljournalname": "Science (New York, N.Y.)",
            "sortdate": "2008/05/16 00:00",
            "pmclivedate": "2009/11/12 00:00"
        }
    }
}