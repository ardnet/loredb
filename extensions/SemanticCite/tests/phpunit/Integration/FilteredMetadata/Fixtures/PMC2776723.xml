<?xml version="1.0"?>
<!DOCTYPE pmc-articleset PUBLIC "-//NLM//DTD ARTICLE SET 2.0//EN" "http://dtd.nlm.nih.gov/ncbi/pmc/articleset/nlm-articleset-2.0.dtd">
<pmc-articleset><article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" article-type="research-article" xml:lang="EN">
  <!--The publisher of this article does not allow downloading of the full text in XML form.-->
  <?properties manuscript?>
  <front>
    <journal-meta>
      <journal-id journal-id-type="nlm-journal-id">0404511</journal-id>
      <journal-id journal-id-type="pubmed-jr-id">7473</journal-id>
      <journal-id journal-id-type="nlm-ta">Science</journal-id>
      <journal-title>Science (New York, N.Y.)</journal-title>
      <issn pub-type="ppub">0036-8075</issn>
      <issn pub-type="epub">1095-9203</issn>
    </journal-meta>
    <article-meta>
      <article-id pub-id-type="pmid">18487186</article-id>
      <article-id pub-id-type="pmc">2776723</article-id>
      <article-id pub-id-type="doi">10.1126/science.1152662</article-id>
      <article-id pub-id-type="manuscript">NIHMS142954</article-id>
      <article-categories>
        <subj-group subj-group-type="heading">
          <subject>Article</subject>
        </subj-group>
      </article-categories>
      <title-group>
        <article-title>Design Logic of a Cannabinoid Receptor Signaling Network That Triggers Neurite Outgrowth</article-title>
      </title-group>
      <contrib-group>
        <contrib contrib-type="author">
          <name>
            <surname>Bromberg</surname>
            <given-names>Kenneth D.</given-names>
          </name>
        </contrib>
        <contrib contrib-type="author">
          <name>
            <surname>Ma'ayan</surname>
            <given-names>Avi</given-names>
          </name>
        </contrib>
        <contrib contrib-type="author">
          <name>
            <surname>Neves</surname>
            <given-names>Susana R.</given-names>
          </name>
        </contrib>
        <contrib contrib-type="author">
          <name>
            <surname>Iyengar</surname>
            <given-names>Ravi</given-names>
          </name>
          <xref rid="FN1" ref-type="author-notes">*</xref>
        </contrib>
        <aff id="A1">Department of Pharmacology and Systems Therapeutics, Mount Sinai School of Medicine, New York, NY 10029, USA</aff>
      </contrib-group>
      <author-notes>
        <corresp id="FN1">
<label>*</label>To whom correspondence should be addressed. <email>ravi.iyengar@mssm.edu</email></corresp>
      </author-notes>
      <pub-date pub-type="nihms-submitted">
        <day>22</day>
        <month>10</month>
        <year>2009</year>
      </pub-date>
      <pub-date pub-type="ppub">
        <day>16</day>
        <month>5</month>
        <year>2008</year>
      </pub-date>
      <pub-date pub-type="pmc-release">
        <day>12</day>
        <month>11</month>
        <year>2009</year>
      </pub-date>
      <volume>320</volume>
      <issue>5878</issue>
      <fpage>903</fpage>
      <lpage>909</lpage>
      <abstract>
        <p id="P1">Cannabinoid receptor ...</p>
      </abstract>
      <contract-num rid="GM1">R01 GM072853-04
				||GM</contract-num>
      <contract-num rid="GM1">R01 GM054508-21
				||GM</contract-num>
      <contract-num rid="GM1">P50 GM071558-030007
				||GM</contract-num>
      <contract-num rid="GM1">P50 GM071558-020007
				||GM</contract-num>
      <contract-num rid="GM1">P50 GM071558-02
				||GM</contract-num>
      <contract-num rid="GM1">P50 GM071558-01A20007
				||GM</contract-num>
      <contract-num rid="GM1">P50 GM071558-01A2
				||GM</contract-num>
      <contract-sponsor id="GM1">National Institute of General Medical Sciences : NIGMS</contract-sponsor>
    </article-meta>
  </front>
</article>
</pmc-articleset>