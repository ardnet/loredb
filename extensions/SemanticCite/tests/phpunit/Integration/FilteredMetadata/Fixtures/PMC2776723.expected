{{#scite:PMC2776723
 |type=research-article
 |title=Design Logic of a Cannabinoid Receptor Signaling Network That Triggers Neurite Outgrowth
 |author=Bromberg KD;Ma'ayan A;Neves SR;Iyengar R|+sep=;
 |journal=Science (New York, N.Y.)
 |pubdate=2008/05/16 00:00
 |year=2009
 |volume=320
 |issue=5878
 |pages=903-909
 |doi=10.1126/science.1152662
 |abstract=Cannabinoid receptor ...
 |pmid=18487186
 |pmcid=PMC2776723
}}